### PR TITLE
[CDAP-14698] Changes 'directives' tab name to 'Transformation steps' in dataprep

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-14698
Build: https://builds.cask.co/browse/CDAP-UDUT202

### CODE REVIEW

File: cdap-ui/app/cdap/text/text-en.yaml [hunk 1]
```diff
@@ -379,7 +379,7 @@ features:
         toggle:
           clearAll: Clear All
           selectAll: Select All
-      directivesTabLabel: Directives ({directivesCount})
+      directivesTabLabel: Transformation steps ({directivesCount})
```
Comment (elfenheart) [inline cdap-ui/app/cdap/text/text-en.yaml]: Will this actually fit in the tab without wrapping?

Comment (ajainarayanan) [reply cdap-ui/app/cdap/text/text-en.yaml]: Yes. It behaves the same way as "Columns" tab on resize.  Do you have suggestions for any changes?

directivestab

Comment (elfenheart) [reply cdap-ui/app/cdap/text/text-en.yaml]: Don't really have a suggestion, it just looked a little weird. If @leacd is good with, then all good!

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Right now on-hold as @bdmogal still deciding on scope of changes we need to do

File: cdap-ui/app/cdap/text/text-en.yaml (hunk 1 above)
Comment (ajainarayanan) [reply cdap-ui/app/cdap/text/text-en.yaml]: 👍 Talked to @leacd. All good

**Comment by ajainarayanan** [comment]
Closing this as this PR is now too old and cannot be rebased against release/6.0. Have opened a new PR for release/6.0: https://github.com/cdapio/cdap/pull/11297

**Pull request closed without merging**